src/codeu/chat/codeU_db/DataBaseConnection.java

```diff
@@ -45,12 +43,21 @@ private void close() {
     }
   }
 
-  public boolean dbUpdate(String str) {
+  public boolean dbUpdate(Vector<String> parameters, String str) {
     open();
+    PreparedStatement stmt = null;
     boolean status = true;
+    int parCounter = 1;
+
     try {
-      stmt = c.createStatement();
-      stmt.executeUpdate(str);
+      stmt = c.prepareStatement(str);
```

Comment:
Since you don't need stmt outside this try block you should put the variable declaration here instead of above. As a general rule it better to put variable declarations as close as possible to their initial usage and minimize the scope they are valid for. This applies to several places below as well. It also results in one less line of code in each instance. :)